### PR TITLE
GOK-167 | Add. ABHA Number and ABHA Address creation

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

**Comment by gsluthra** [comment]
Is there a screenshot? And a JIRA issue?

File: app/src/main/res/layout/activity_auth_mode.xml
Comment (sudiptaDuttaTW) [inline app/src/main/res/layout/activity_auth_mode.xml:1]: please remove unused code from xml.

**Comment by SanoferSameera** [comment]
> Is there a screenshot? And a JIRA issue?

trello issue: https://trello.com/c/NeRDYSkh/167-abha-number-abha-address-creation-using-aadhaar-otp-flow

File: app/src/main/res/layout/activity_auth_mode.xml
Comment (SanoferSameera) [reply app/src/main/res/layout/activity_auth_mode.xml:1]: done

**Pull request merged**